### ISSUE TITLE
react-bootstrap or fela?

### ISSUE THREAD

**Issue by veeramarni** [js-accounts/react#43]
Do you have plans to expand to another popular styling such as react-bootstrap and fela?  We could contribute in that area and can send pr's if interested.

**Comment by TimMikeladze** [comment]
Yes we absolutely have plans to support other themes. We're planning a redesign of the code right now in #42 which will provide us with a concrete approach for handling different themes. 

If you could send a PR once #42 is completed that would be fantastic and greatly appreciated.

**Comment by veeramarni** [comment]
Ok great! Do you guys plan to use typescript? I was asking because it would also help #42

**Comment by TimMikeladze** [comment]
Using Typescript is something I want to use for the core accounts package but I haven't considered using it for the react package. However it's probably best to have it all be in TS.
